Dockerfile

```diff
@@ -1,36 +1,145 @@
-# This image is based on the official sonar-scanner image
-FROM sonarsource/sonar-scanner-cli:4.4
+# Builder image for other analysis tools
+FROM debian:10.5-slim AS builder
+
+# Get sonar-scanner and C/C++ tools sources
+ADD https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-4.4.0.2170.zip \
+    https://downloads.sourceforge.net/project/cppcheck/cppcheck/1.90/cppcheck-1.90.tar.gz \
+    https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/rough-auditing-tool-for-security/rats-2.4.tgz \
+    http://downloads.sourceforge.net/project/expat/expat/2.0.1/expat-2.0.1.tar.gz \
+    /
+
+# Compile CppCheck from source
+RUN apt-get update \
```

Comment:
Codacy found an issue: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/gh/cnescatlab/sonar-scanner/pullRequest?prid=6081579)